rust/Cargo.lock

```diff
@@ -23,7 +23,7 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
I would remove the Cargo.lock update from this PR, we can update it separately later.